zaplib/cargo-zaplib/src/build_npm_package.rs

```diff
@@ -0,0 +1,58 @@
+use std::process::Command;
+use log::info;
+
+pub async fn build_npm_package(path: String) {
```

Comment:
```suggestion
pub async fn build_npm_package(path: &str) {
```

then you don't need to clone at the call site. #nit